src/footer/_footer.scss

```diff
@@ -0,0 +1,192 @@
+@include govuk-exports("footer") {
+
+  $govuk-footer-background: $govuk-grey-3;
+  $govuk-footer-border-top: #a1acb2;
+  $govuk-footer-border: $govuk-grey-2;
+  $govuk-footer-text: #454a4c;
+  $govuk-footer-link: $govuk-footer-text;
+  $govuk-footer-link-hover: #171819;
+
+  // Based on the `govuk-file-url("govuk-crest.png");` image dimensions.
+  $govuk-footer-crest-image-width-2x: 250px;
+  $govuk-footer-crest-image-height-2x: 204px;
+  // Half the 2x image so that it fits the regular 1x size.
+  $govuk-footer-crest-image-width: ($govuk-footer-crest-image-width-2x / 2);
+  $govuk-footer-crest-image-height: ($govuk-footer-crest-image-height-2x / 2);
+
+  .govuk-c-footer {
+    @include govuk-font-regular;
+    @include govuk-typography-responsive($govuk-font-16);
+
+    padding-top: $govuk-spacing-scale-7;
```

Comment:
I think these should probably be using the responsive spacing scale

`@include govuk-responsive-padding($govuk-spacing-responsive-7, "top");`
`@include govuk-responsive-padding($govuk-spacing-responsive-5, "bottom");`